Formula/qemu.rb

```diff
@@ -34,6 +34,7 @@ class Qemu < Formula
   depends_on "nettle"
   depends_on "pixman"
   depends_on "snappy"
+  depends_on "spice-protocol"
```

Comment:
Good point. I'll try to add it as a configure option only. I consider a test to be difficult. In this particular case it would be to see if the clipboard functionality works afterwards in a virtual instance created with QEMU.

Reply:
```
Last 15 lines from /Users/brew/Library/Logs/Homebrew/qemu/01.configure:
Run-time dependency xkbcommon found: NO (tried pkgconfig)
Run-time dependency slirp found: YES 4.7.0
Has header "libvdeplug.h" : YES 
Library vdeplug found: YES
Run-time dependency jack found: NO (tried pkgconfig)
Run-time dependency sndio found: NO (tried pkgconfig)
Run-time dependency spice-protocol found: NO (tried pkgconfig)
Run-time dependency spice-server found: NO (tried pkgconfig)

../meson.build:749:10: ERROR: Dependency "spice-server" not found, tried pkgconfig
```

Does not work if a only add `--enable-spice`. The solution provided works. It adds the spice protocol if the dependency is correctly set.

Reply:
What about 
```ruby
depends_on "spice-protocol" => :build
```
?

Reply:
Thanks for the suggestion and help. I will try if this works.

Reply:
Successfully tested and pushed with the new modifications.